apps/shipments/migrations/0035_historical_device_shipments.py

```diff
@@ -0,0 +1,145 @@
+                ('customs_hold_date_act', models.DateTimeField(blank=True, null=True)),
+                ('customs_release_date_act', models.DateTimeField(blank=True, null=True)),
+                ('container_type', models.CharField(blank=True, max_length=255, null=True)),
+                ('port_arrival_locode', models.CharField(blank=True, max_length=255, null=True)),
+                ('final_port_locode', models.CharField(blank=True, max_length=255, null=True)),
+                ('import_locode', models.CharField(blank=True, max_length=255, null=True)),
+                ('lading_locode', models.CharField(blank=True, max_length=255, null=True)),
+                ('origin_locode', models.CharField(blank=True, max_length=255, null=True)),
+                ('us_routed', models.CharField(blank=True, max_length=255, null=True)),
+                ('import_customs_mode', models.CharField(blank=True, max_length=255, null=True)),
+                ('us_export_port', models.CharField(blank=True, max_length=255, null=True)),
+                ('version', models.CharField(default='1.2.0', max_length=255)),
+                ('trailer_number', models.CharField(blank=True, max_length=255, null=True)),
+                ('seal_number', models.CharField(blank=True, max_length=255, null=True)),
+                ('is_master_bol', models.BooleanField(default=False)),
+                ('nmfc_class', models.CharField(blank=True, max_length=4, null=True, validators=[django.core.validators.RegexValidator(message='Invalid nmfc class number.', regex='^1[015]0$|^1[27]5$|^2[05]0$|^[345]00$|^[567]0$|^[568]5$|^92\\.5$|^77\\.5$')])),
+                ('is_hazmat', models.BooleanField(default=False)),
+                ('customer_fields', django.contrib.postgres.fields.jsonb.JSONField(blank=True, null=True)),
+                ('history_date', models.DateTimeField()),
+                ('history_change_reason', models.CharField(max_length=100, null=True)),
+                ('history_type', models.CharField(choices=[('+', 'Created'), ('~', 'Changed'), ('-', 'Deleted')], max_length=1)),
+                ('history_id', models.CharField(default=apps.utils.random_id, max_length=36, primary_key=True, serialize=False)),
+                ('bill_to_location', models.ForeignKey(blank=True, db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='shipments.Location')),
+                ('device', models.ForeignKey(blank=True, db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='shipments.Device')),
+                ('final_destination_location', models.ForeignKey(blank=True, db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='shipments.Location')),
+                ('history_user', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to=settings.AUTH_USER_MODEL)),
```

Comment:
How does this work? We don't really have a User model in Transmission (technically we do but it's only used for the admin panel). We would need to use something like this: https://github.com/treyhunner/django-simple-history/blob/master/simple_history/models.py#L296

Reply:
Also see this: https://django-simple-history.readthedocs.io/en/2.7.0/user_tracking.html#manually-track-user-model